### PR TITLE
[ES-1125] Cherrypicked fixes from develop to release-1.0.x

### PR DESCRIPTION
**Pull request by balaji-alluru**

### CODE REVIEW

File: signup-service/pom.xml [hunk 1]
```diff
@@ -115,7 +115,7 @@
 		<dependency>
 			<groupId>io.mosip.esignet</groupId>
 			<artifactId>esignet-core</artifactId>
-			<version>1.3.0-SNAPSHOT</version>
+			<version>1.4.0</version>
```
Comment (ase-101) [inline signup-service/pom.xml:118]: this should be 1.4.1-SNAPSHOT

File: signup-service/pom.xml [hunk 2]
```diff
@@ -172,7 +172,7 @@
 		<dependency>
 			<groupId>io.mosip.esignet</groupId>
 			<artifactId>esignet-integration-api</artifactId>
-			<version>1.3.0-SNAPSHOT</version>
+			<version>1.4.0</version>
```
Comment (ase-101) [inline signup-service/pom.xml:175]: this should be 1.4.1-SNAPSHOT

File: signup-service/pom.xml (hunk 1 above)
Comment (balaji-alluru) [reply signup-service/pom.xml]: updated the same as suggested

File: signup-service/pom.xml (hunk 2 above)
Comment (balaji-alluru) [reply signup-service/pom.xml]: updated the same as suggested

**Review by ase-101 — APPROVED** [review]

**Pull request merged**